src/lib/Handset/CRSFHandset.cpp

```diff
@@ -268,14 +265,23 @@ void CRSFHandset::RcPacketToChannelsData() // data is packed as 11 bits per chan
         bitsMerged -= srcBits;
     }
 
-    if (prev_AUX1 != ChannelData[4])
-    {
-        #if defined(PLATFORM_ESP32)
-        devicesTriggerEvent();
-        #endif
-    }
+    //
+    // sends channel data and also communicates commanded armed status in arming mode Switch.
+    // frame len 24 -> arming mode CH5: use channel 5 value
+    // frame len 25 -> arming mode Switch: use commanded arming status in extra byte
+    //
+    armCmd = inBuffer.asUint8_t[1] == 24 ? CRSF_to_BIT(ChannelData[4]) : payload[readByteIndex];
+
+    #if defined(PLATFORM_ESP32)
```

Comment:
I think the `#if` is required just around the `devicesTriggerEvent();` call, otherwise the 8285 RX-as-TX is not going to work correctly with the new arming.

Reply:
done